lib/compat/wordpress-6.0/blocks.php

```diff
@@ -122,3 +122,36 @@ function gutenberg_build_query_vars_from_query_block( $block, $page ) {
 	}
 	return $query;
 }
+
+if ( ! function_exists( 'extend_block_editor_settings_with_discussion_settings' ) ) {
+	/**
+	 * Workaround for getting discussion settings as block editor settings
+	 * so any user can access to them without needing to be an admin.
+	 *
+	 * @param array $settings Default editor settings.
+	 *
+	 * @return array Filtered editor settings.
+	 */
+	function extend_block_editor_settings_with_discussion_settings( $settings ) {
```

Comment:
It's a filter so it won't be backported the same way. It doesn't need to be guarded with a function call. It's engough to prefix the function name with `gutenberg_`.

Reply:
Done!